src/main/java/com/ays/backend/user/exception/DeviceNotException.java

```diff
@@ -0,0 +1,11 @@
+package com.ays.backend.user.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class DeviceNotException extends RuntimeException {
```

Comment:
please fix the typo
```suggestion
public class DeviceNotFoundException extends RuntimeException {
```

Reply:
I just revised the class name as DeviceNotFoundException instead of DeviceNotException